specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-03-preview/managedClusters.json

```diff
@@ -3522,10 +3522,46 @@
         "windowsProfile": {
           "$ref": "#/definitions/AgentPoolWindowsProfile",
           "description": "The Windows agent pool's specific profile."
+        },
+        "networkProfile": {
+          "$ref": "#/definitions/AgentPoolNetworkProfile",
+          "description": "Network-related settings of agent pools."
```

Comment:
minor: Possibly phrase this as: "Network-related settings of an agent pool"

Reply:
done